css/signin.css

```
@@ -101,8 +101,19 @@ input:focus {
     right: 15px;
     top: 50%;
     transform: translateY(-50%);
+    cursor: pointer;
 }
 
+.error-message {
+    color: #FF5B56;
```

```suggestion
    color: var(~~정의하신이름);
```
색깔 요소들이 정의되어 있다면 활용하는게 차후 유지보수할 때 좋습니다!
색깔이 만약에 일괄적으로 수정되어야 한다면, 변수로 처리해두었을 때는 사용하는 곳 일일히 수정하지 않아도 되기 떄문이죵